lxd/operations.go

```diff
@@ -561,7 +590,12 @@ func operationsGet(d *Daemon, r *http.Request) response.Response {
 	err = s.DB.Cluster.Transaction(context.TODO(), func(ctx context.Context, tx *db.ClusterTx) error {
 		var err error
 
-		membersWithOps, err = tx.GetNodesWithOperations(ctx, projectName)
+		switch allProjects {
```

Comment:
Generally in the codebase we use `if/else` for regular boolean checks:

```go
if allProjects {

} else {

}
```

Reply:
got it. And the commit for such change starts with `lxd/operations` too, right?

Reply:
That seems fine yes.